rspace/src/main/scala/coop/rchain/rspace/history/HistoryInstances.scala

```diff
@@ -341,27 +342,140 @@ object HistoryInstances {
           elements.lastOption.map(t => (remainingPath, t))
         )
 
+      def processSubtree(
+          start: (Blake2b256Hash, Option[LastModification])
+      )(elem: (Byte, List[HistoryAction])): F[(Byte, (Blake2b256Hash, Option[(KeyPath, Trie)]))] =
```

Comment:
Can you add some type aliasses explaining the meaning of passed arguments?

Also, in cases where we do a match immediately (like here) - wouldn't it make sense to have a dedicated case class?